src/components/profile/ProfileService.js

```diff
@@ -256,11 +269,16 @@ goog.require('ga_urlutils_service');
         };
 
         this.get = function(feature) {
+          if (!feature || !feature.getGeometry()) {
+            var defer = $q.defer();
+            defer.resolve(emptyData);
+            return defer.promise;
+          }
           var coordinates = feature.getGeometry().getCoordinates();
 
           // TODO: manage all kind of geometry
```

Comment:
Maybe in the future in case we allow modification of external KML. It's just a reminder.